.github/workflows/cicd.yaml

```diff
@@ -30,7 +30,7 @@ jobs:
     env:
       OS: linux
       SQL_PYSDK_ENABLE: OFF
-      SQL_JAVASDK_ENABLE: OFF
+      SQL_JAVASDK_ENABLE: ON # enable to build and package Java files
```

Comment:
There is no need to build java package expect release on tag branch

Reply:
Yes. But we have to set the environment here so that the Makefile can generate right Cmake parameters then we can `make install` with Java packages.

Reply:
Compared with base line https://github.com/4paradigm/OpenMLDB/actions/runs/1610101387, `cpp_ut_test` takes about 30 minutes longer now. Can we add task-manager building task to `release` stage or find other method

Reply:
might solve by add a step:
```yaml
- name: enable java on release
   if: github.event_name == 'push'
   run: |
     echo "SQL_JAVASDK_ENABLE=ON" >> $GITHUB_ENV
```